node_modules/gh-events/lib/internal/dao.js

```diff
@@ -158,6 +158,81 @@ var updateEvent = module.exports.updateEvent = function(event, update, callback)
 };
 
 /**
+ * Update the organisers of an event
+ *
+ * @param  {Event}      event               The event to update
+ * @param  {Object}     update              The updates to persist. Keys that are numbers are considered to be references to users, anything else is considered to be a plain-text organiser
```

Comment:
I think it would be nicer to have them split off here (maybe even split them off in the REST API)